ykcompile/src/lib.rs

```diff
@@ -598,6 +600,21 @@ impl<TT> TraceCompiler<TT> {
         Location::Register(reg)
     }
 
+    fn c_len(&mut self, p: &Place) -> Location {
+        let dst = self.create_temporary();
+        let (loc, _) = self.place_to_location(p, true);
+        match loc {
+            Location::Register(src) | Location::Addr(src) => {
+                dynasm!(self.asm
+                    ; mov Rq(dst), [Rq(src) + 8]
```

Comment:
This is access into a fat pointer, right? Should we comment this?

Reply:
Yes, I was just thinking that too.

Reply:
Try this: 119bbb